### DISCUSSION TITLE
World map with downloaded countries/areas

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#686] [Q&A] [answered]
In the past there were already dicussions about a world map at low zoom levels. What do you think about an integrated world map with basic content (borders, rivers, relief data) and showing the areas of downloaded maps and with different color of activated maps as an overlay? At higher zoom levels the "normal" map data should be displayed instead of the basic world map.

**Comment by devemux86** [comment, marked as answer]
@kds112 Thanks for the suggestion.

There is already an existing thread on this topic:

- https://github.com/devemux86/cruiser/discussions/101
